sdk/src/main/java/pt/ua/dicoogle/sdk/datastructs/dim/Serie.java

```diff
@@ -1,31 +1,31 @@
 /**
  * Copyright (C) 2014  Universidade de Aveiro, DETI/IEETA, Bioinformatics Group - http://bioinformatics.ua.pt/
  *
- * This file is part of Dicoogle/dicoogle.
+ * This file is part of Dicoogle/dicoogle-sdk.
  *
- * Dicoogle/dicoogle is free software: you can redistribute it and/or modify
+ * Dicoogle/dicoogle-sdk is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
  *
- * Dicoogle/dicoogle is distributed in the hope that it will be useful,
+ * Dicoogle/dicoogle-sdk is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with Dicoogle.  If not, see <http://www.gnu.org/licenses/>.
  */
-package pt.ua.dicoogle.core.dim;
+package pt.ua.dicoogle.sdk.datastructs.dim;
 
 import java.net.URI;
 import java.util.ArrayList;
 
 /**
  *
- * @author Luís A. Bastião Silva <bastiao@ua.pt>
+ * @author Luís A. Bastião Silva <bastiao@bmd-software.com>
  */
-public class Serie
+public class Serie implements SerieInterface
```

Comment:
This could be an opportunity to fix the typo: `Serie` to `Series`, and `SerieInterface` to `SeriesInterface`. Will leave that to you.